config/detekt/detekt-override.yml

```
@@ -1,6 +1,37 @@
 complexity:
   active: true
   #
+  # ComplexMethod
+  #
+  # 概要
+  # - メソッドの複雑度具合
+  #
+  # 補足
+  # - 1 つのメソッドに以下が含まれるごとに複雑度が 1 追加される
+  #  - Conditional statements - if, else if, when
+  #  - Jump statements - continue, break
+  #  - Loops - for, while, do-while, forEach
+  #  - Operators &&, ||, ?:
+  #  - Exceptions - catch, use
+  #  - Scope Functions - let, run, with, apply, and also ->Reference
+  #
+  ComplexMethod:
+    active: true
+    threshold: 16 # (default: 16)
```

[Must]

デフォルト値：16 -> 15

でお願いします

```suggestion
    threshold: 16 # (default: 15)
```

デフォルト値は15のようだったので

https://github.com/sunakan/realworld-kotlin-springboot-jdbc/blob/f5fcbca133cbd4704d2761cf13df11738dad6173/config/detekt/detekt.yml#L98-L103

---

こちら修正しました。

https://github.com/sunakan/realworld-kotlin-springboot-jdbc/pull/97/commits/9b95942e64d8384130150dd66b7591423236564c